src/controllers/issuer.ts

```diff
@@ -13,7 +13,7 @@ import { generateDidDoc, validateSpecCompliantPayload } from '../helpers/helpers
 export class IssuerController {
 	public static createValidator = [
 		check('didDocument')
-			.optional()
+			.optional({nullable: true})
```

Comment:
Why do we need it here? As I know we already allowed to pass empty DIDDocument, did we?

Reply:
Done! ✅